src/definitions/modules/shape.js

```diff
@@ -759,7 +759,15 @@ $.fn.shape = function(parameters) {
         if(instance === undefined) {
           module.initialize();
         }
-        module.invoke(query);
+        var $inputs = $module.find('input');
+        if( $inputs.length > 0) {
+          $inputs.blur();
+          setTimeout(function(){
```

Comment:
Why is this timeout needed?

Reply:
Without the timeout the browser does still have the information that any input is still focused and still breaks the animation, even a value  below 100ms did not work. Try it out without the timeout and you will recognize it won't work

Reply:
Ok that's fine, was going to suggest changing to `0` but if you have tested that its fine 👍